src/report/postmark.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@nestjs/common';
+import * as postmark from 'postmark';
+import { CreateReportDto } from './dto';
+import { TemplatedMessage } from 'postmark';
+import * as process from 'process';
+import { MessageSendingResponse } from 'postmark/dist/client/models';
+
+@Injectable()
+export class PostmarkService {
+  private client: postmark.ServerClient;
+
+  constructor() {
+    this.client = new postmark.ServerClient(process.env['POSTMARK_API_TOKEN']!);
+  }
+
+  async sendBarkBeetleReport(
+    createReportDto: CreateReportDto,
+    images: Array<Express.Multer.File>,
+    to: string,
+  ): Promise<MessageSendingResponse> {
+    function encodeToBase64(file: Express.Multer.File) {
+      return Buffer.from(file.buffer).toString('base64');
+    }
+
+    const attachments = [];
+
+    for (let i = 0; i < images.length; i++) {
+      const encodedImageContent = encodeToBase64(images[i]);
+      attachments.push({
+        Name: 'nuotrauka' + i + '.jpg',
+        Content: encodedImageContent,
+        ContentType: 'image/jpeg',
```

Comment:
The image won't always be 'image/jpeg'. I would recommended to detect the file type and extension dynamically using a magic number approach. Utilize libraries like `file-type` (https://www.npmjs.com/package/file-type) or equivalent for detection.